editor/plugins/gdextension/cpp_scons/template/addon/gdext.gitignore

```diff
@@ -0,0 +1,10 @@
+bin/
```

Comment:
Please expand with entries from the slightly larger [godot-cpp-template .gitignore](https://github.com/godotengine/godot-cpp-template/blob/main/.gitignore).

Reply:
This file contains ignores that can either be inserted into the root (in the case of modules) or within the GDExtension addon (in the case of GDExtensions by themselves). In the former case we need to ensure we don't ignore `*.obj` so I have that handled separately. Some entries in there are redundant with what's already expected to be in a Godot project's root, such as `.godot/` and `.DS_Store`. IDE folder ignores should also go in the project's root separately from the GDExtension. Also `.sconsign*.dblite` is not needed when `*.dblite` is ignored. But thanks, I've applied a bunch of things from here, will push later.